lib/kaiser/plugin.rb

```diff
@@ -35,6 +35,8 @@ def self.loaded?(name)
     def self.inherited(plugin)
       super
 
+      puts "INHERITED #{plugin}"
```

Comment:
Is this for temporary debugging? Was this line intended to be committed to the git repo?

Reply:
o yes. will remove